assets/src/components/home/Home.tsx

```diff
@@ -13,13 +13,19 @@ import { ClusterOverviewCard } from './clusteroverview/ClusterOverviewCard'
 import { DeploymentsCard } from './deployments/DeploymentsCard'
 
 import { PrCard } from './pullrequests/PrCard'
+import { ConstraintViolationsCard } from './managerview/violations/ConstraintViolationsCard'
 
 const breadcrumbs: Breadcrumb[] = [{ label: 'home', url: '/' }]
 
 export default function Home() {
   useSetBreadcrumbs(breadcrumbs)
   const { me } = useContext(LoginContext)
 
+  const isManager =
```

Comment:
we should probably make some hooks for this (in case they'll be needed to reconfigure other ui elements)